packages/app/src/cli/models/app/extensions.ts

```diff
@@ -35,6 +35,7 @@ export const UIExtensionConfigurationSchema = schema.define.object({
 
   // Only for CustomerAccountsUiExtension
   categories: schema.define.array(schema.define.string()).optional(),
+  localization: schema.define.any().optional(),
```

Comment:
how does this work for checkout ui extensions?
It should somehow already be in place, shouldn't it?

Reply:
that is so weird/wild to me but hey

Reply:
@TobiasBales I think your sentence after `but hey` got cut off?

Reply:
no sorry, weird phrase I guess
it was something like "but hey, what do I know"